README.md

```diff
@@ -27,3 +27,196 @@ for installing dependencies, you can refer to the CI scripts in this repo:
 
 - [install-debian-deps.sh](.github/scripts/install-debian-deps.sh) (Debian, Ubuntu, Mint, etc.)
 - [install-macos-deps.sh](.github/scripts/install-macos-deps.sh) (macOS with Homebrew)
+
+## Windows Usage Caveats
+
+Windows support is currently WIP. The following current "quirks" apply:
+
+- It will default to WASAPI and use your primary output device. Ability to select different devices or audio drivers (ASIO, etc..) is not yet supported.
+- When pasting in multiline strings, they should work. Just note that many built-in Windows terminals by default will strip out certain characters like tabs,
+    ruining your beautiful formatting. This can be changed in the terminal's settings.
+
+## Windows Build + Development
+
+(I'm new to this toolchain so some things may be wrong)
+
+Windows support is achieved by building via [msys2](https://www.msys2.org/) (using mingw-w64-ucrt6) as opposed to building natively on Windows (probably possible but probably much more annoying).
+
+I believe the project specifically requires clang.
```

Comment:
i made it gcc compatible too, at least on Linux. does it not work on WIndows?

Reply:
this was entirely based on a discord message I saw from you about forcing clang, and me not trying anything else. I can give gcc a try and update this!

Reply:
ah right, that was early on when we still depended on libdispatch which is clang-only

Reply:
just checked and it's working fine on gcc on Windows!